couchpotato/core/media/movie/providers/info/couchpotatoapi.py

```diff
@@ -102,12 +106,141 @@ def getInfo(self, identifier = None, adding = False, **kwargs):
 
         return {}
 
+
     def getReleaseDate(self, identifier = None):
         if identifier is None: return {}
 
         dates = self.getJsonData(self.urls['eta'] % identifier, headers = self.getRequestHeaders())
-        log.debug('Found ETA for %s: %s', (identifier, dates))
 
+        #This grabs release date info from omdbapi/rottentomatoes
+        temp2 = self.getJsonData("http://www.omdbapi.com/?i=%s&tomatoes=true&plot=short&r=json" % identifier)
+        title = temp2['Title']
+        year = int(temp2['Year'])
+
+        ddate=0 #throw away what couchpotatoai is returning since it is garbage at this time
+        tdate=0
+        dvd_date= temp2['DVD']
+        theater_date=temp2['Released']
+        if theater_date != 'N/A' and year >1972:
+            p='%d %b %Y'
+            tdate=int(time.mktime(time.strptime(theater_date,p)))
+
+        if dvd_date != 'N/A' and year >1972:
+            p='%d %b %Y'
+            ddate=int(time.mktime(time.strptime(dvd_date,p)))
+           
+        if (ddate !=0):    
+            if ddate < tdate:
+                ddate = 0 #if the dvd release date occurs before the theater release date, assume the data is wrong
+                tdate = 0
+
+        if (ddate or tdate):
```

Comment:
hopefully your case doesn't fall into that "merging" info situation..

this would be an issue if OMDBAPI had only theater release date...

THEMOVIEDB had only dvd release date..

in this case, THEMOVIEDB would be checked, but the set of data returned from OMDBAPI would be used... (i.e. the theater release date would be used and the dvd release date would be discarded - its not that theater takes precedence over dvd, but that OMDBAPI takes precedence over THEMOVIEDB -- as it is implemented now).. 

it would be quite easy to add some kind of merging situation.. if one service has a dvd_date and the other has a theater_date.. then, make sure they are sane (i.e. theater_date < ddvd_date) and use them both instead of thowing one away..  I'll try that out righ tnow and commit - so dont bother testing the changes just yet.